pkg/maps/ctmap/lookup.go

```diff
@@ -20,83 +20,9 @@ import (
 	"strconv"
 
 	"github.com/cilium/cilium/pkg/bpf"
-	"github.com/cilium/cilium/pkg/byteorder"
-	"github.com/cilium/cilium/pkg/tuple"
 	"github.com/cilium/cilium/pkg/u8proto"
 )
 
-// CtKey4 is needed to provide CtEntry type to Lookup values
```

Comment:
Please don't delete huge swathes of code and place it into another file in the same commit as functional changes.